Compiling with NOHQ is broken

Maybe a similar issue as in #2705?

```
$ cmake -DMUPENPLUSAPI=On -DNOHQ=On ../../src/ -G Ninja
$ cmake --build .
[4/116] Generate Git version
current revision "fb645bfc"
last build revision
[116/116] Linking CXX shared library mupen64plus-video-GLideN64.dll
FAILED: mupen64plus-video-GLideN64.dll libmupen64plus-video-GLideN64.dll.a
C:\WINDOWS\system32\cmd.exe /C "cd . && C:\msys64\ucrt64\bin\c++.exe -std=c++11 -pthread -O3 -DNDEBUG -flto=auto -fno-fat-lto-objects  -static-libgcc -O3 -shared -o mupen64plus-video-GLideN64.dll -Wl,--out-implib,libmupen64plus-video-GLideN64.dll.a -Wl,--major-image-version,0,--minor-image-version,0 @CMakeFiles\mupen64plus-video-GLideN64.rsp && cd ."
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:\msys64\tmp\ccCTxqMq.ltrans4.ltrans.o:<artificial>:(.text+0x3f42): undefined reference to `txfilter_checksum_strong'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:\msys64\tmp\ccCTxqMq.ltrans4.ltrans.o:<artificial>:(.text+0x406f): undefined reference to `txfilter_checksum_strong'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:\msys64\tmp\ccCTxqMq.ltrans4.ltrans.o:<artificial>:(.text+0x7261): undefined reference to `txfilter_dmptx_strong'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:\msys64\tmp\ccCTxqMq.ltrans4.ltrans.o:<artificial>:(.text+0x813b): undefined reference to `txfilter_dmptx_strong'
C:/msys64/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/14.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:\msys64\tmp\ccCTxqMq.ltrans4.ltrans.o:<artificial>:(.text+0x8787): undefined reference to `txfilter_dmptx_strong'
collect2.exe: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
```

--- a/src/TxFilterStub.cpp
+++ b/src/TxFilterStub.cpp
@@ -31,8 +31,20 @@ txfilter_checksum(uint8 *src, int width, int height, int size, int rowStride, ui
 	return 0U;
 }
 
+TAPI uint64 TAPIENTRY
+txfilter_checksum_strong(uint8 *src, int width, int height, int size, int rowStride, uint8 *palette)
+{
+	return 0U;
+}
+
 TAPI boolean TAPIENTRY
 txfilter_dmptx(uint8 *src, int width, int height, int rowStridePixel, uint16 gfmt, N64FormatSize n64FmtSz, Checksum r_crc64)
+{
+	return 0;
+}
+
+TAPI boolean TAPIENTRY
+txfilter_dmptx_strong(uint8 *src, int width, int height, int rowStridePixel, uint16 gfmt, N64FormatSize n64FmtSz, Checksum r_crc64)
 {
 	return 0;
 }

